assets/src/stories-editor/components/right-click-menu/index.js

```diff
@@ -100,16 +208,53 @@ const RightClickMenu = ( props ) => {
 				{
 					name: __( 'Duplicate Block', 'amp' ),
 					blockAction: duplicateBlock,
+					params: [ firstBlockClientId ],
 					icon: 'admin-page',
 					className: 'right-click-duplicate',
 				},
 			);
 		}
 
+		const pageList = getBlockOrder();
+		const pageNumber = pageList.length;
+		if ( block && pageNumber > 1 ) {
+			const currentPage = getCurrentPage();
+			const currentPagePosition = pageList.indexOf( currentPage );
+			if ( currentPagePosition > 0 ) {
+				const prevPage = getPageByOffset( -1 );
+				if ( isBlockAllowedOnPage( block.name, prevPage ) ) {
+					blockActions.push(
+						{
+							name: __( 'Send block to previous page', 'amp' ),
```

Comment:
While we're at it, I'd like to replace usage of the word "block" with "element" in this component. We should be more strict there.

```suggestion
							name: __( 'Move element to previous page', 'amp' ),
```

Reply:
As other items in the list already use block and that is like in 1.3, that is out of scope of this ticket. Please following up with another issue.